### DISCUSSION TITLE
Prebuilds - set `CI=true` or non-interactive mode

### DISCUSSION THREAD

**Discussion by aaronadamsCA** [community/community#16561] [Codespaces]
Can you please run prebuilds actions in a non-interactive shell, or add `CI=true` to the environment?

Prebuilds actions are currently logging output that is only appropriate for an interactive shell, which makes an absolute mess of the logs. They are massively long and very difficult to read.

**Comment by sharma-anuradha** [comment]
Thank you @aaronadamsCA , for reaching out. Can you please share the example GitHub Action log with this issue?

**Reply by aaronadamsCA** [reply]
Here you go: https://github.com/aaronadamsCA/codespaces-prebuilds-issue/runs/6509096378?check_suite_focus=true

![image](https://user-images.githubusercontent.com/1505561/169317603-fc7bc8b7-9bd6-4b47-a064-0aa6dbee0448.png)

Compare to a different repository that uses `CI=true` and you'll notice a significant difference:

![image](https://user-images.githubusercontent.com/1505561/169319010-1bb5f68e-8a2f-4ff0-9550-c1029d344c21.png)

**Reply by sharma-anuradha** [reply]
Thank you for the logs :). We will take it from here ....

**Reply by shyim** [reply]
Have the same problem the logs are soo long and cant even find anything

**Reply by aaronadamsCA** [reply]
@sharma-anuradha, Codespaces prebuilds still seem to run in interactive mode, making logs a lot longer and harder to read than they need to be; any progress here?
